functions/api/v1/notifications/[id].ts

```diff
@@ -48,7 +48,7 @@ export async function handleRequest(
 	const fromAccount = await loadExternalMastodonAccount(acct, fromActor)
 
 	const out: Notification = {
-		id: row.notif_id.toString(),
+		id: (row.notif_id as object).toString(),
```

Comment:
Similar issue as `mastodon/notification.ts`

Reply:
again, I've applied a temporary solution and will try to see if a better type and be defined and shared 👍

Reply:
updated, please have a look 🙂